### PR TITLE
zar-1-set-up-orm

### PR DESCRIPTION
**Pull request by Sidanton6006**

### CODE REVIEW

File: Domain/Domain/Entities/UserEntity.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Entities
+{
+    internal class UserEntity : IdentityUser<Guid>, IEntity
```
Comment (byCodeXp) [inline Domain/Domain/Entities/UserEntity.cs]: Must be public

File: Domain/Domain/Entities/RoleEntity.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Entities
+{
+    internal class RoleEntity: IdentityRole<Guid>, IEntity
```
Comment (byCodeXp) [inline Domain/Domain/Entities/RoleEntity.cs]: Must be public

File: Domain/Domain/Entities/Abstract/IEntity.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Entities.Abstract
+{
+    internal interface IEntity
```
Comment (byCodeXp) [inline Domain/Domain/Entities/Abstract/IEntity.cs]: Must be public

File: Domain/Domain/Entities/Abstract/BaseEntity.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Entities.Abstract
+{
+    internal class BaseEntity: IEntity
```
Comment (byCodeXp) [inline Domain/Domain/Entities/Abstract/BaseEntity.cs]: Must be public

File: Domain/Domain/ApplicationDbContext.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain
+{
+    internal class ApplicationDbContext : IdentityDbContext<UserEntity, RoleEntity, Guid>
```
Comment (byCodeXp) [inline Domain/Domain/ApplicationDbContext.cs]: Must be public

**Review by byCodeXp — CHANGES_REQUESTED** [review]
Add Infrastructure layer and move there ApplicationDbContext.
All entities classes must be public

File: Core/Entities/Abstract/BaseEntity.cs [hunk 6]
```diff
@@ -0,0 +1,9 @@
+﻿namespace Core.Entities.Abstract
+{
+    public class BaseEntity:IEntity
```
Comment (byCodeXp) [inline Core/Entities/Abstract/BaseEntity.cs]: Need to be abstract

**Review by byCodeXp — APPROVED** [review]
Seems good!

**Review by Mike-Opanasiuk — APPROVED** [review]

**Pull request merged**